docs/pages/Configuration.md

```diff
@@ -702,6 +706,20 @@ Version | UI5 CLI Release
 **1.0** | v1.0.0+
 **0.1** | v0.0.1+
 
+### Specification Version 3.0
```

Comment:
Also add a note here that v3 is currently in beta. Ideally linking to the migration page.